recipe/build.sh

```diff
@@ -22,4 +33,4 @@ else
 fi
```

Comment:
This should ensure we always add PTX support at the end of the list

Also might be a good idea to `echo` this value at the end (to ensure it is what we expect)

```suggestion
  export TORCH_CUDA_ARCH_LIST="${TORCH_CUDA_ARCH_LIST}+PTX"
  echo "TORCH_CUDA_ARCH_LIST=${TORCH_CUDA_ARCH_LIST}"
fi
```